js/lib/confirmation-modal.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @package Polylang
+ */
+
+// We can't use underscore or lodash in this common code because it depends of the context classic or block editor.
+// Classic editor underscore is loaded, Block editor lodash is loaded.
+const { __ } = wp.i18n;
+
+const languagesList = jQuery( '#post_lang_choice');
+
+export const bypassConfirmation = () => {
+	let title = jQuery( 'input#title' ).val();
+	let content = jQuery( 'textarea#content' ).val();
+	let excerpt = jQuery( 'textarea#excerpt' ).val();
+	// Block editor case
+	if ( document.getElementById( 'editor' ) && 'undefined' !== typeof wp.data ) {
+		const editor = wp.data.select( 'core/editor' );
+		title = editor.getEditedPostAttribute( 'title' ).trim();
+		content = editor.getEditedPostAttribute( 'content' ).trim();
+		excerpt = editor.getEditedPostAttribute( 'excerpt' ).trim();
+	}
+	return ! title && ! content && ! excerpt;
+}
+
+// Dialog box for alerting the user about a risky changing.
+export const initializeConfimationModal = () => {
+	// Create dialog container.
+	const dialogContainer = jQuery(
+		'<div/>',
+		{
+			id: 'pll-dialog',
+			style: 'display:none;'
+		}
+	).text( __( 'Are you sure you want to change the language of the current content?', 'polylang' ) );
```

Comment:
Thankfully, this looks pretty similar to the [React.createElement()](https://reactjs.org/docs/react-without-jsx.html) signature. So sharing the internationalized strings should be simple when we decide to do it.